cookbooks/bcpc-hadoop/attributes/default.rb

```diff
@@ -5,10 +5,10 @@
 #############################################
 require 'pathname'
 
-default['bcpc']['hadoop'] = {}
-default['bcpc']['hadoop']['distribution']['release'] = '2.3.6.2-3'
-default['bcpc']['hadoop']['distribution']['active_release'] = node['bcpc']['hadoop']['distribution']['release']
-default['bcpc']['hadoop']['decommission']['hosts'] = []
+default["bcpc"]["hadoop"] = {}
```

Comment:
thanks for catching this.  i ran rubocop and foodcritic in my libraries but totatlly skipped the attributes i changed.